src/commons/stores/volume-store.js

```diff
@@ -61,8 +61,10 @@
                         profileStatus = "Pending";
                     } else if (data[i].profiling_enabled === "yes") {
```

Comment:
rather then if-else can we write a JSON and do it?
something like:
{"no": "Disabled",
"yes": "Enabled",
"unknown": "Unknown"}

Reply:
@cloudbehl We are sure of "yes" and "no" value only. Apart from that 3rd value can be blank/null/none. Hence, need to do this way.